src/templates/layouts/vamcSystem/index.test.tsx

```diff
@@ -196,7 +196,7 @@ describe('VamcSystem with valid data', () => {
       expect(vaLink).toBeInTheDocument()
       expect(vaLink).toHaveAttribute('href', `${mockData.path}/locations`)
       expect(vaLink).toHaveAttribute('text', 'See all locations')
-      expect(vaLink).toHaveAttribute('active', 'true')
+      expect(vaLink).toHaveAttribute('active', '')
```

Comment:
I think this is due to a change in how React 19 handles boolean values for DOM attributes. I double-checked that it's working properly in the browser, and it's golden.